influxdb/src/lib.rs

```diff
@@ -56,17 +56,18 @@
 //!         }.into_query("weather"),
 //!     );
 //!
-//!     let write_result = client
-//!         .query(weather_readings)
-//!         .await;
-//!     assert!(write_result.is_ok(), "Write result was not okay");
+//!     if let Err(e) = client.query(weather_readings).await {
+//!         println!("Error writing result: {e}");
+//!         return;
+//!     }
```

Comment:
Why not use `.expect()`? I mean we don't actually run the doc tests, but if we did, it would fail using assert/expect/panic/... but not when using println and return